admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -162,7 +162,7 @@ pupilRestartDataService.sqlMarkRestartAsDeleted = async (restartId, userId) => {
 
                    -- IF the new check is not in NEW status we have to bail out
                    IF @newCheckCode <> 'NEW'
-                    THROW 5100, 'New check cannot be deleted as it does not have a NEW status', 1;
+                    THROW 51000, 'New check cannot be deleted as it does not have a NEW status', 1;
```

Comment:
have you tested this on azure, as sp_addmessage does not exist and you should verify the behaviour of custom message codes on sql azure.  should 'just work' or you may have to use 50000.  not ideal, but thats how they do it

Reply:
This fixes an issue as the error code needs to >= 51000. Looks like a 0 got missed off.